index.html

```diff
@@ -160,18 +160,18 @@ <h3 class="name-project">project name goes here</h3>
               <li class="lang-project">Ruby on rails</li>
               <li class="lang-project">Javascript</li>
             </ul>
-            <button class="button-project">
+            <a class="button-project" id="btn4-1" href="#header12">
               See this project
               <span
                 ><img src="./assests/images/Union.png" alt="arrow image"
               /></span>
-            </button>
-            <button class="button-project768">
+            </a>
+            <a class="button-project768" id="btn4-2" href="#header12">
               See this project
               <span
                 ><img src="./assests/images/Union2.png" alt="arrow image"
               /></span>
-            </button>
+            </a>
```

Comment:
- [ ] Please already created HTML code for the "projects list" should be removed, and instead, it should be added via JavaScript.
Kindly follow the explanations I wrote below